src/index.js

```diff
@@ -49,10 +48,10 @@ class WebSockets {
     }
 
     return multiaddrs.filter((ma) => {
-      if (includes(ma.protoNames(), 'ipfs')) {
-        ma = ma.decapsulate('ipfs')
-      }
-      return mafmt.WebSockets.matches(ma) || mafmt.WebSocketsSecure.matches(ma)
+      return mafmt.WebSockets.matches(ma) ||
+        mafmt.WebSocketsSecure.matches(ma) ||
+        mafmt.WebSocketsIPFS.matches(ma) ||
+        mafmt.WebSocketsSecureIPFS.matches(ma)
```

Comment:
As for being able to just do IPFS.match(ma) as you pointed out in https://github.com/whyrusleeping/js-mafmt/pull/16#discussion_r133625774, I don't think that would work, since it would also filter TCP and WebRTC addrs here.